onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -228,6 +228,14 @@ def parse_arguments(argv: Optional[List[str]] = None) -> argparse.Namespace:
     )
     model_group.set_defaults(vocab_mask=False)
 
+    model_group.add_argument(
+        "--past_present_share_buffer",
+        required=False,
+        action="store_true",
+        help="Use kv_cache for past and present, currently work for gpt2 greedy search.",
+    )
+    model_group.set_defaults(past_present_share_buffer=False)
```

Comment:
Why not enable it by default?

Reply:
First to backward compatible. We could change it later.